### ISSUE TITLE
Unable to deploy wasm contracts with newer rust version

### ISSUE THREAD

**Issue by shariffdev** [near/near-workspaces-rs#294]
After the merge of PR #293 , which was expected to resolve the issue mentioned in [nearcore issue #9143](https://github.com/near/nearcore/issues/9143), it seems that the problem persists when using the latest main hash as a dependency. This issue aims to track the investigation and resolution of this persistent error.

**Comment by mrLSD** [comment]
**It's more context about the issue:**

Example PR: aurora-is-near/release-repository/pull/2

It was updated toolchain from `nightly-2023-03-15` to `stable`.
And it was `cargo update`.

Tests fail with a `Deserialization` error.

cc: @shariffdev

**Comment by mrLSD** [comment]
@shariffdev any progress on that issue?

**Comment by itsyaasir** [comment]
I apologize for the delay in getting back to you. I have conducted a more in-depth investigation into the issue raised in the [PR](https://github.com/aurora-is-near/release-repository/pull/2). During this analysis, I encountered some unusual behavior; however, I was able to successfully execute the tests, which seem to be functioning properly now and address the [logging ](https://github.com/near/workspaces-rs/issues/272)issue you mentioned earlier.

As an interim solution, I managed to devise a workaround that appears to be stable for the time being. Here are the steps I took to rectify the issue:

- Compile the NEAR sandbox from the source (or you can actually retrieve the sandbox inside the release-repository after building the project should be here -> release-repository/target/debug/build/near-sandbox-utils-xxxx/out/near-sandbox)
- Run your tests using the compiled sandbox.

Please note, this is a temporary solution and not a final resolution. I wanted to ensure that the tests could run without any hitches as I look more into it.

You can run the test using the following command:
```sh
NEAR_SANDBOX_BIN_PATH="./near-sandbox"  cargo t
```
I have attached my screenshot below and as you can see; it silences the log messages as well.

![Screenshot 2023-09-09 at 7 07 23 PM](https://github.com/near/workspaces-rs/assets/7762687/717e434c-b4b8-487d-a5b1-ee58ccd8bb84)

**Comment by shariffdev** [comment]
@mrLSD 

I apologize for the delay in getting back to you. I have conducted a more in-depth investigation into the issue raised in the [PR](https://github.com/aurora-is-near/release-repository/pull/2). During this analysis, I encountered some unusual behavior; however, I was able to successfully execute the tests, which seem to be functioning properly now and address the [logging ](https://github.com/near/workspaces-rs/issues/272)issue you mentioned earlier.

As an interim solution, I managed to devise a workaround that appears to be stable for the time being. Here are the steps I took to rectify the issue:

- Compile the NEAR sandbox from the source (or you can actually retrieve the sandbox inside the release-repository after building the project should be here -> release-repository/target/debug/build/near-sandbox-utils-xxxx/out/near-sandbox)
- Run your tests using the compiled sandbox.

Please note, this is a temporary solution and not a final resolution. I wanted to ensure that the tests could run without any hitches as I look more into it.

You can run the test using the following command:
```sh
NEAR_SANDBOX_BIN_PATH="./near-sandbox"  cargo t
```
I have attached my screenshot below and as you can see; it silences the log messages as well.

![Screenshot 2023-09-09 at 7 07 23 PM](https://github.com/near/workspaces-rs/assets/7762687/717e434c-b4b8-487d-a5b1-ee58ccd8bb84)

**Comment by mrLSD** [comment]
Tjanja. When do you planing release solution, when things works out of the box?

**Comment by shariffdev** [comment]
@mrLSD A fix is in the process currently, I will notify you when the fix is out.

**Comment by shariffdev** [comment]
@mrLSD We have a [PR](https://github.com/near/near-workspaces-rs/pull/311) in progress which should be able to fix the problems mentioned in the PR.

**Comment by shariffdev** [comment]
This issue has been resolved, you can wait for 0.8.0 release or you can use latest commit. I am closing this issue

**Comment by mrLSD** [comment]
Thanks @shariffdev !
Is there any schedule for the release date?

**Comment by shariffdev** [comment]
Hey @mrLSD a new version has been released, you are good to update to 0.8.0
